WooCommerce/Classes/ViewRelated/Products/Cells/ProductsTabProductTableViewCell.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+final class ProductsTabProductTableViewCell: UITableViewCell {
```

Comment:
As mentioned in a previous PR, this should have a nib file with the subviews in interface builder.